worksheet_A/TerminalHacking.cpp

```diff
@@ -37,10 +85,49 @@ int main()
 	for each (std::string word in options)
 	{
 		std::cout << word << std::endl;
+		
 	}
 
-	// TODO: implement the rest of the game
+	// Main game loop
+	while (livesRemaining > 0)
+	{
+		std::cout << "Please input a guess using only capital letters." << std::endl;
+		std::cin >> playerGuess;
+		for each (std::string word in options)
+		{
+			if (playerGuess != word)
```

Comment:
This will only allow the player to guess the last word in the list! You shouldn't be setting `validWord = false` inside the loop. Will also work if you `break` out of the loop after setting `validWord = true`

However note that there is an easier (and more computationally efficient) way to check whether a `std::set` contains a given element; see http://stackoverflow.com/questions/1701067/how-to-check-that-an-element-is-in-a-stdset